packages/firestore/src/local/memory_persistence.ts

```diff
@@ -129,6 +132,15 @@ export class MemoryPersistence implements Persistence {
     return this.indexManager;
   }
 
+  getDocumentOverlay(user: User): DocumentOverlayCache {
```

Comment:
This should end in "Cache" (in Android as well). Otherwise it sounds like we are returning a single overlay.